backend/unitTests/chatAttackTest.py

```
@@ -0,0 +1,243 @@
+        print("ERROR: Cannot connect to server")
+        return False
+
+def test_sql_injection_via_chat_text():
+    """Test SQL injection through chat text - SECURITY VULNERABILITY"""
+    chat_data = {
+        "username": "test_user",
+        "text": "Show me my items'; DROP TABLE items; SELECT 'hacked"
+    }
+    
+    try:
+        response = requests.post(f"{BASE_URL}/api/chat", json=chat_data)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)